client/blocks/product-purchase-features-list/business-onboarding.jsx

```diff
@@ -13,16 +13,23 @@ import { noop } from 'lodash';
  */
 import PurchaseDetail from 'components/purchase-detail';
 
-export default localize( ( { translate, link, onClick = noop } ) => {
+export default localize( ( { isBusinessPlan, translate, link, onClick = noop } ) => {
```

Comment:
My understanding was that `isBusinessPlan` has been the name for targeting WordPress.com Business plan sites throughout the codebase, and I wanted to maintain so as well. 

But, given that the purpose of this code is to differentiate the copy between WordPress.com and Jetpack sites, I think naming it `isWPcomPlan` makes better sense. 👍🏽

Reply:
Yeah, I've seen both Jetpack Professional and WP.com business plans being referred to as `business`, so `isWpcomPlan` mitigates the confusion. I understand why you named it `isBusinessPlan` too, and I'd also be good with it in case it's your preference - consistency always makes sense. Up to you really 👍